lib/hooks/gitlab/templates/note.html.haml

```diff
@@ -0,0 +1,38 @@
+:ruby
+  note            = payload.object_attributes
+  has_url         = note.url?
+  has_user        = payload.user?
+  repository_name = has_url ? note.url.split('/', 5)[-2] : '' # http://example.com/owner/:name/issues/:id
+  has_issue       = payload.issue?
+  has_merge       = payload.merge_request?
```

Comment:
I think this abbreviation is meaningless.
`has_merge_request` should be better.

Reply:
@kaznum 
changed ``has_merge_request``
```
:ruby
  note                = payload.object_attributes
  has_url             = note.url?
  has_user            = payload.user?
  repository_name     = has_url ? note.url.split('/', 5)[-2] : '' # http://example.com/owner/:name/issues/:id
  has_issue           = payload.issue?
  has_merge_request   = payload.merge_request?
  has_snippet         = payload.snippet?
  has_commit          = payload.commit?

```